cluster/mesos/docker/deploy-addons.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
In the current state the whole docker-compose setup for k8sm is for fire-and-forget test clusters. Upgrades of addons doesn't play a role here (yet). I would prefer to defer any change to this part to a later PR. I have added a ticket: https://github.com/mesosphere/kubernetes-mesos/issues/424